_docs/2.0.0/build-process.md

```diff
@@ -4,4 +4,5 @@ title: Build Process
 version: '2.0.0'
 categories:
     - '2.0.0'
+redirect_from: /docs/build-process
```

Comment:
What does this do?

Reply:
It does the redirect you wanted, is it not?

So `URL/docs/build-process` and `URL/docs/LATEST-VERSION/build-process` is the same

Reply:
Ah so it instructs the build to create the redirection from RL/docs/build-process toURL/docs/LATEST-VERSION/build-process? Cool 👍